### PR TITLE
Added quickstart #10 (obj-c)

### PR DESCRIPTION
**Pull request by volbap**
- Added quickstart 10-Customizing-Lock.
- Also, updated `index.yml` adding that one.

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by sebacancinos** [comment]
+1 @mpaktiti @ntotten

**Comment by mpaktiti** [comment]
Sequence to merge obj-c quickstarts:
1. Sample for User Profile https://github.com/auth0-samples/auth0-ios-objc-sample/pull/5
2. Quickstart for User Profile https://github.com/auth0/docs/pull/1724
3. Sample for Linking Accounts https://github.com/auth0-samples/auth0-ios-objc-sample/pull/6
4. Quickstart for Linking Accounts https://github.com/auth0/docs/pull/1775
5. Sample for Rules https://github.com/auth0-samples/auth0-ios-objc-sample/pull/8
6. Quickstart for Rules https://github.com/auth0/docs/pull/1818/files
7. Sample for Authorization https://github.com/auth0-samples/auth0-ios-objc-sample/pull/9
8. Quickstart for Authorization https://github.com/auth0/docs/pull/1819
9. Sample for Customizing Lock https://github.com/auth0-samples/auth0-ios-objc-sample/pull/7
10. [THIS PR] Quickstart for Customizing Lock

File: articles/native-platforms/ios-objc/10-customizing-lock.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+A0Theme *myAwesomeTheme = [[A0Theme alloc] init];
+// Configure your theme here (see step 2)
+[[A0Theme sharedInstance] registerTheme: myAwesomeTheme];
+```
+
+### 2. Configure your Theme
+
+Lock's widget UI is composed of several parts that can be customized. Here is a list of the available values that you can modify:
+
+![Lock.png](/media/articles/libraries/lock-ios/customization/Lock-UI-Parts.png)
```
Comment (mpaktiti) [inline articles/native-platforms/ios-objc/10-customizing-lock.md:36]: @pablolvillar Please resize this image to max 750 pixels wide.

Also, according to our [guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#borders) a border should be added: _For close-ups and other screenshots that do not include the browser window, apply a 1px centered border (select all > edit > stroke) of color `#cccccc` to keep the image from blending with the background and appearing to float._

File: articles/native-platforms/ios-objc/10-customizing-lock.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+You can configure the following type of properties:
+
+- **Color**: `UIColor` instance.
+- **Image**: `String` with image name.
+- **Font**: `UIFont` instance.
+
+For example:
+
+```
```
Comment (mpaktiti) [inline articles/native-platforms/ios-objc/10-customizing-lock.md:48]: @pablolvillar This should be `swift` maybe? There is no highlighting as is.

File: articles/native-platforms/ios-objc/10-customizing-lock.md (hunk 1 above)
Comment (volbap) [inline articles/native-platforms/ios-objc/10-customizing-lock.md:36]: Thanks for the "photoshop for dummies"-like instructions 😄

Comment (volbap) [inline articles/native-platforms/ios-objc/10-customizing-lock.md:36]: Fixed.

**Comment by volbap** [comment]
@mpaktiti Done with the fixes.

**Comment by mpaktiti** [comment]
Before deployment:
- [Seed project](https://github.com/auth0-samples/auth0-ios-objc-sample/pull/7) should be merged. The URL in this page should be updated (currently points to invalid sample).
- Sample is at auth0-samples repo, download zip functionality should be in place.
- `index.yml` should be checked again

File: articles/native-platforms/ios-objc/10-customizing-lock.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../includes/github', { link: 'https://github.com/auth0-samples/auth0-ios-objc-sample/tree/master/10-Customizing-Lock', }) %>
```
Comment (mpaktiti) [inline articles/native-platforms/ios-objc/10-customizing-lock.md]: @pablolvillar @sebacancinos 
Both the directory and the filename are wrong and this causes the page to fail to load. The correct is `../../_includes/_github`

**Comment by volbap** [comment]
@mpaktiti I fixed the sample project link. This one is ready.

**Comment by mpaktiti** [comment]
Content moved here: https://github.com/auth0/docs/pull/2328

**Pull request closed without merging**